services/webnn/webnn_switches.h

```diff
@@ -43,10 +43,12 @@ inline constexpr char kWebNNOrtUseOpenvino[] = "webnn-ort-use-openvino";
 inline constexpr char kWebNNOrtDisableCpuFallback[] =
     "webnn-ort-disable-cpu-fallback";
 
-// For GPU, OV EP will use FP16 inference precision by default, this switch will
-// force OV EP to use FP32 inference precision to get better accuracy, but it
-// may result in decreased performance.
-inline constexpr char kWebNNOrtUseOVGpuFP32[] = "webnn-ort-use-ov-gpu-fp32";
+// For GPU, OV EP will use FP16 inference precision by default. You can specify
+// the flag to use FP32 or ACCURACY inference precision to get better accuracy,
+// but it may result in decreased performance.
+// Usage1: --webnn-ort-ov-gpu-precision=FP32
+// Usage2: --webnn-ort-ov-gpu-precision=ACCURACY
```

Comment:
You may want to explain what the underlying setting to OV execution mode when specifying "FP32" or "ACCURACY". Adding a pointer to OV documentation would be helpful.